### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by emlynjdavies**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (tankefugl) [inline .github/ISSUE_TEMPLATE/bug_report.md:31]: Is this part needed? I don't think these are very relevant for PySilCam, or?

**Comment by tankefugl** [comment]
I think having templates is good! However, I think that having them as minimal as possible and as large as necessary makes them more relevan and easier to use them. I suggest taking a small "can we remove this/do we really need this?"-check on the items in the templates to make them as PySilCam-relevant as possible.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (emlynjdavies) [reply .github/ISSUE_TEMPLATE/bug_report.md:31]: True. I clicked the template button and github made the pull request immediately :)

Comment (emlynjdavies) [reply .github/ISSUE_TEMPLATE/bug_report.md]: the issue template should be more suitable now

**Pull request merged**